nix/mobile/android/build.nix

```diff
@@ -106,6 +108,7 @@ in stdenv.mkDerivation rec {
     export ANDROID_NDK_ROOT="${androidPkgs.ndk}"
 
     export STATUS_NIX_MAVEN_REPO="${deps.gradle}"
+    export RCT_METRO_PORT=${toString reactMetroPort}
```

Comment:
I don't think you need this, the setting of `RCT_METRO_PORT` in the context of derivation arguments should be enough.

Reply:
If we are setting `-PreactNativeDevServerPort` then why do we even need the `RCT_METRO_PORT` env var for?

Reply:
Yeah good point, I tried removing these extra `RCT_METRO_PORT` variables and everything seems to be working well.